documentation/asciidoc/stories/assembly_configuring_client_certificates.adoc

```diff
@@ -5,7 +5,7 @@ ifdef::context[:parent-context: {context}]
 
 [role="_abstract"]
 Add client trust stores to your project and configure {brandname} to allow connections only from clients that present valid certificates.
-This increases security of your deployment by ensuring that clients are trusted by a public certificate authority (CA).
+This increases security of your deployment by ensuring that clients are trusted by a public certificate authority (CA).    
```

Comment:
why this is still in the PR as there is only  a whtiespace change

Reply:
+1 for the content

Reply:
Whitespace change should be reverted before we merge.

Reply:
Removed the white space.